XMTPiOSExample/XMTPiOSExample/Views/LoggedInView.swift

```diff
@@ -8,15 +8,21 @@
 import SwiftUI
 import XMTP
 
+class EnvironmentCoordinator: ObservableObject {
+	@Published var path = NavigationPath()
+}
+
 struct LoggedInView: View {
 	var client: XMTP.Client
 
+	@StateObject var environmentCoordinator = EnvironmentCoordinator()
+
 	var body: some View {
-		NavigationView {
+		NavigationStack(path: $environmentCoordinator.path) {
```

Comment:
Using `NavigationStack` instead of `NavigationPath` here means the demo app is iOS 16 only. Feels worth it to me since it's just for educational purposes and using the old navigation stuff ends up with piles of hacks on piles of hacks.